imap_processing/hit/l0/decom_hit.py

```diff
@@ -357,7 +357,11 @@ def assemble_science_frames(sci_dataset: xr.Dataset) -> xr.Dataset:
     # Convert sequence flags and counters to NumPy arrays for vectorized operations
     seq_flgs = sci_dataset.seq_flgs.values
     seq_ctrs = sci_dataset.src_seq_ctr.values
-    science_data = sci_dataset.science_data.values
+    # TODO: improve this as needed
+    binary_str_val = []
+    for data in sci_dataset.science_data.values:
+        binary_str_val.append("".join(f"{byte:08b}" for byte in data))
+    science_data = binary_str_val
```

Comment:
Just curious why this update is needed. Did changes to space packet parser result in returning a list of bytes instead of a full binary string?

Reply:
@vmartinez-cu  Yes, now it returns a list of bytes

Reply:
yes. you got it right. If your data is broken down in 8bits chunks, then you can use return value of `space_packet_parser` directly. But for HIT case, I think we do need to convert bytes back to binary strings since your unpacking is little complicated.